src/codegen/general.rs

```diff
@@ -24,6 +27,29 @@ pub fn start_comments(w: &mut Write, conf: &Config) -> Result<()> {
     Ok(())
 }
 
+pub fn start_comments_no_version(w: &mut Write) -> Result<()> {
+    try!(writeln!(
+        w,
+        "// This file was generated by gir (https://github.com/gtk-rs/gir)
+// DO NOT EDIT"));
```

Comment:
Can you add `from gir-files (https://github.com/gtk-rs/gir-files)` as well please?

Reply:
Done